plugins/vendor/zoom/lib/model/oAuth/kZoomOauth.php

```diff
@@ -134,11 +134,26 @@ public static function verifyHeaderToken($zoomConfiguration)
 				return;
 			}
 
-			KalturaLog::debug("Zoom signature header received [$signatureHeader]");
-			KalturaLog::debug("Zoom request timestamp header received [$signatureTimestamp]");
-			KalturaLog::debug("Signature created for comparison [$signature]");
+			KalturaLog::debug("Zoom signature mismatch: request timestamp [$signatureTimestamp],
+							request signature [$signatureHeader], calculated signature [$signature]");
 		}
 
 		ZoomHelper::exitWithError(kZoomErrorMessages::FAILED_VERIFICATION);
 	}
+
+	/**
+	 * @param array $zoomConfiguration
+	 * @return string
+	 */
+	public static function getSecretTokenForEncryption($zoomConfiguration)
```

Comment:
change the name of the function to getTokenForEncryption, calling it with secret in the name is confusing since you can return either secret or validate

Reply:
done